src/server.ts

```diff
@@ -41,43 +39,32 @@ export interface OperationMessage {
   type: string;
 }
 
-export type ExecuteReactiveFunction = (
-  schema: GraphQLSchema,
-  document: DocumentNode,
-  rootValue?: any,
-  contextValue?: any,
-  variableValues?: {[key: string]: any},
-  operationName?: string,
-) => IObservable<ExecutionResult>;
-
-export type ExecuteFunction = (
-  schema: GraphQLSchema,
-  document: DocumentNode,
-  rootValue?: any,
-  contextValue?: any,
-  variableValues?: {[key: string]: any},
-  operationName?: string,
-) => Promise<ExecutionResult>;
-
-export type SubscribeFunction = (
-  schema: GraphQLSchema,
-  document: DocumentNode,
-  rootValue?: any,
-  contextValue?: any,
-  variableValues?: {[key: string]: any},
-  operationName?: string,
-) => AsyncIterator<ExecutionResult>;
-
-export interface Executor {
-  execute?: ExecuteFunction;
-  executeReactive?: ExecuteReactiveFunction;
-  subscribe?: SubscribeFunction;
-}
+export type ExecuteFunction = (schema: GraphQLSchema,
+                               document: DocumentNode,
+                               rootValue?: any,
+                               contextValue?: any,
+                               variableValues?: { [key: string]: any },
+                               operationName?: string) => Promise<ExecutionResult> | AsyncIterator<ExecutionResult>;
```

Comment:
@leebyron I wanted to make sure that anyone who uses this package and expects graohql-js's execute will get exactly that. It almost won't effect you and the docs will be focused on that. But at the same time, if you want to use a separate, experimental package, it won't block you from it.